src/seedu/addressbook/data/person/Person.java

```diff
@@ -88,4 +89,43 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Comparator to sort via name.
+     */
+    public static Comparator<Person> compPersonName = new Comparator<Person>() {
+        @Override
+        public int compare(Person p1, Person p2) {
+            return p1.getName().toString().compareTo(p2.getName().toString());
```

Comment:
This implementation is alright for now, but moving forward, do think about how this could be improved. Especially with regards to splitting the work across the `Person` and `Name` classes.